Gemfile

```diff
@@ -1,8 +1,8 @@
 source 'https://rubygems.org'
 
+gem 'fastlane', '~> 2.166'
 gem 'cocoapods', '~> 1.9'
-gem 'fastlane', '~> 2.1'
 gem 'xcode-install', '~> 2.6'
 
 plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
-eval_gemfile(plugins_path) if File.exist?(plugins_path)
+eval_gemfile('fastlane/Pluginfile') if File.exist?(plugins_path)
```

Comment:
@EwoutH could you clarify this change?

Reply:
Of course, it fixes this issue with Dependabot: https://github.com/dependabot/dependabot-core/issues/1720#issuecomment-600831687

Reply:
Thanks!